unfoldingWord/translationCore
# Only show tW part 1 in non-developer mode

#### This pull request addresses:
- #2145 

#### How to test this pull request:
- Select a project and go then go t the tools page and only `tW` part 1 should render on the screen.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2240)
